backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3725,4 +3725,51 @@ databaseChangeLog:
             GRANT SELECT ON ${database.defaultSchemaName}.result TO ${noPhiUsername};
       rollback:
         sql: |
-          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
\ No newline at end of file
+          ALTER TABLE ${database.defaultSchemaName}.result DROP COLUMN test_result;
+  - changeSet:
+      id: add-test-correction-columns
+      author: nathan@skylight.digital
+      comment: Adds columns in support of test correction workflow
+      changes:
+        - tagDatabase:
+            tag: add-test-correction-columns
+        - addColumn:
+            tableName: test_order
+            columns:
+              - column:
+                  name: corrected_by
+                  type: uuid
+                  constraints:
+                    nullable: true
+                    foreignKeyName: fk__test_order__api_user
+                    references: api_user(internal_id)
+              - column:
+                  name: corrected_at
+                  type: DATETIME
+                  constraints:
+                    nullable: true
+              - column:
+                  name: additional_correction_information
```

Comment:
I think it probably makes the most sense to leave the existing `Reason_For_Correction` as-is.  And just create a new column for the enum value.  All of your naming options are kind of gross because of how many options are already taken by existing columns.  Half tempted to just recommend `Correction_Enum` but I that feels at least a little gross

Reply:
Or a two phase migration.  Create `correction_reason` and the other fields you have listed here.  Populate `additional_correction_information` with data from `reason_for _correction`. Once everything is updated to point to `additional_correction_information` delete the old `reason_for_correction` field so you don't have two columns with essentially the same name

Reply:
I like this, especially the two-phase approach. See [3d3a348](https://github.com/CDCgov/prime-simplereport/pull/3650/commits/3d3a3481f5d7b03dcbe71339b33617615db212d8) for the first step of this. It will be a little depressing to have both a `correction_reason` and a `reason_for_correction` temporarily, but that really is the best name for our enum-holding column.